students_list.py

```diff
@@ -12,32 +12,35 @@
 
 class StudentsList:
     def __init__(self, class_name):
+        """ :param str class_name: name of the class """
```

Comment:
You can elaborate `name of the class` a bit more as it can be confused with [python's class](https://docs.python.org/3/tutorial/classes.html)

Reply:
Sure! Nice catch! Is writing 
```python
""" :param str class_name: name of the physical class of a student"""
```
sufficient? 
Please suggest.
Thanks!